docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: "3.0"
+services:
+  kazan:
+    build: .
+    volumes:
+      - ./:/usr/app
```

Comment:
I usually prefer to not introduce `docker-compose` if there are no additional services required to run a docker container with the application. Docker Compose was build exactly to simplify deployment of multiple services. For just one docker container, it introduces unnecessary complexity.

Reply:
I had same thought on adding  ```docker-compose.yml``` . My take was if a developer is not familiar with IT Operational Tools/Docker it can go with single command 
And as purpose of docker here is for development  **Bind Mount** are necessary and some people might struggle with
binding local directory   with docker directory as there issues like OS/ space in names of directory

if ```docker-compose.yml```  is not there we have remove docker compose instructions and add basic guide like ```-v $(pwd):/use/app```for Linux   in Redmi.md